src/ICSharpCode.SharpZipLib/Zip/ZipEntryExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ICSharpCode.SharpZipLib.Zip
+{
+	/// <summary>
+	/// General ZipEntry helper extensions
+	/// </summary>
+	public static class ZipEntryExtensions
```

Comment:
Does it need to be public? (not sure if it's useful for other users or not?)

Reply:
Maybe not, but I know that I would appreciate having it when consuming the library (since the flag is exposed in a public property as an `int`).